README.md

```diff
@@ -53,23 +53,48 @@ This gem provides a Rails-aware [haml](http://haml.info/) renderer. This rendere
     = root_path
     ```
 
-## Quick Start
+## Installation
+
+### Add the gem to your app
 
 1. Add the following to your Rails application's `Gemfile`:
 
     ```ruby
-    gem "txi_rails_hologram"
+    gem "txi_rails_hologram", github: "tablexi/txi_rails_hologram"
```

Comment:
should we recommend `git: "https://github.com/tablexi/txi_rails_hologram.git"` as per security-recommendations from `bundle-audit`?